components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/dao/impl/PolicyDAOImpl.java

```diff
@@ -2032,6 +2046,24 @@ public Policy getSimplifiedPolicyByLevelAndName(APIMgtAdminService.PolicyLevel p
                             policy = new SubscriptionPolicy(resultSet.getString(APIMgtConstants
                                     .ThrottlePolicyConstants.COLUMN_UUID), resultSet.getString(APIMgtConstants
                                     .ThrottlePolicyConstants.COLUMN_NAME));
+                            setCommonPolicyDetails(policy, resultSet);
+                            setSubscriptionPolicyDetals((SubscriptionPolicy) policy, resultSet);
+                            InputStream binary = resultSet.getBinaryStream(APIMgtConstants.ThrottlePolicyConstants.
+                                    COLUMN_CUSTOM_ATTRIB);
+                            if (binary != null) {
+                                byte[] customAttrib;
+                                try {
+                                    customAttrib = IOUtils.toByteArray(binary);
+                                    if (customAttrib.length > 0) {
+                                        ((SubscriptionPolicy) policy).setCustomAttributes(customAttrib);
+                                    }
+                                } catch (IOException e) {
+                                    String errorMsg = "An Error occurred while retrieving custom attributes for " +
+                                            "subscription policy with identifier: " + policy.getUuid();
```

Comment:
uuid in the policy would not give the end-user any meaning. But having a policy name might. Was there any specific reason to put the uuid instead of policy name here?

Reply:
No any specific reason.
I can change that to policy name instead.